lib/rollbax/client.ex

```diff
@@ -162,21 +171,43 @@ defmodule Rollbax.Client do
       Logger.error("(Rollbax) unexpected API status: #{code}/#{description}")
     end
 
-    %{state | hackney_responses: Map.put(responses, ref, [])}
+    %{state | hackney_responses: Map.put(responses, ref, {code, []})}
   end
 
-  defp handle_hackney_response(_ref, {:headers, headers}, state) do
+  defp handle_hackney_response(ref, {:headers, headers}, %{hackney_responses: responses} = state) do
     Logger.debug("(Rollbax) API headers: #{inspect(headers)}")
-    state
+
+    with %{^ref => {429, _body}} <- responses,
```

Comment:
I assume it is possible that we get several of the rate limited responses before we end up here, right? Ideally, we should schedule rate limiting disabling only once.